packages/frontend/core/src/utils/popup.ts

```diff
@@ -1,5 +1,6 @@
 import { DebugLogger } from '@affine/debug';
 import { apis } from '@affine/electron-api';
+import { Browser } from '@capacitor/browser';
 
 const logger = new DebugLogger('popup');
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +15]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->We need to abstract this into a popupWindowProvider<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8581?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->